compiler/AST/build.cpp

```diff
@@ -1714,13 +1697,13 @@ BlockStmt* buildVarDecls(BlockStmt* stmts, std::set<Flag> flags, const char* doc
     Expr* varCount = stmts->blockInfoGet()->get(2);
     tuple->var->defPoint->insertAfter(
       buildIfStmt(new CallExpr("!=", new CallExpr(".", tuple->remove(),
-                                                  new_StringSymbol("size")),
+                                                  new_CStringSymbol("size")),
                                varCount->remove()),
-                  new CallExpr("compilerError", new_StringSymbol("tuple size must match the number of grouped variables"), new_IntSymbol(0))));
+                  new CallExpr("compilerError", new_CStringSymbol("tuple size must match the number of grouped variables"), new_IntSymbol(0))));
 
     tuple->var->defPoint->insertAfter(
       buildIfStmt(new CallExpr("!", new CallExpr("isTuple", tuple->copy())),
-                  new CallExpr("compilerError", new_StringSymbol("illegal tuple variable declaration with non-tuple initializer"), new_IntSymbol(0))));
+                  new CallExpr("compilerError", new_CStringSymbol("illegal tuple variable declaration with non-tuple initializer"), new_IntSymbol(0))));
```

Comment:
I think they probably could use new_StringSymbol, but compilerError looks like:
```chpl
proc compilerError(param x:c_string ...?n) {
  __primitive("error", (...x));
}
```
so the string will be turned right back into a c_string. `__primitive` calls have no type checking or coercion, so passing a `string` into them can lead to errors.